### PR TITLE
[Bug] Fix native scene can't get UUID bug

### PR DESCRIPTION
**Pull request by moshuying**
Re: #https://github.com/cocos/3d-tasks/issues/16828

### Changelog

* Fix native scene can't get UUID bug

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by dumganhar** [comment]
This bug is from 3.7.1 : https://github.com/cocos/cocos-engine/pull/13937/files#diff-e0e3c5fa93d1b1486b6d7ff9ea44f98f13b00624074d754b5417dedb5b4cd0a5

asset uuid getter API was added for web but wasn't added for JSB.

**Review by dumganhar — APPROVED** [review]

File: native/tools/swig-config/assets.i [hunk 1]
```diff
@@ -126,6 +126,7 @@
 //  4. 'Attribute Section' should be placed before 'Import Section' and 'Include Section'
 //
 %attribute(cc::Asset, ccstd::string&, _uuid, getUuid, setUuid);
+%attribute(cc::Asset, ccstd::string&, uuid, getUuid, setUuid);
```
Comment (SantyWang) [inline native/tools/swig-config/assets.i]: uuid only has a getter, no setter.

Comment (dumganhar) [reply native/tools/swig-config/assets.i]: Good finding, we need to remove setUuid at this line.

Comment (moshuying) [reply native/tools/swig-config/assets.i]: OK，setUuid has been removed。

**Review by SantyWang — APPROVED** [review]

**Pull request merged**